buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -964,6 +964,10 @@ private void createConfiguration() {
         // Don't wait for state, just start up quickly. This will also allow new and old nodes in the BWC case to become the master
         defaultConfig.put("discovery.initial_state_timeout", "0s");
 
+        // TODO: Remove these once https://github.com/elastic/elasticsearch/issues/46091 is fixed
+        defaultConfig.put("logger.org.elasticsearch.action.support.master", "DEBUG");
```

Comment:
this won't give us the desired logging, as the logger is non-static and the specific subclasses of TransportMasterNodeAction are in a different package

Reply:
Make it `static` and live with the change in classname in the log lines? (we get that information from logged exceptions anyway?)

Reply:
As a temporary change, yes. As a more permanent change, I'm not so sure (need to think about it more).

Reply:
https://github.com/elastic/elasticsearch/pull/46363/commits/fc7842a1d8a10c0b78f6262dae9abb4815d7cf1e